pallets/elections/src/lib.rs

```diff
@@ -7,25 +7,25 @@
 //! - committee ([`EraValidators`]): Set of nodes that produce and finalize blocks in the session.
 //! - validator: Node that can become a member of committee (or already is) via rotation.
 //! - `EraValidators::reserved`: immutable validators, ie they cannot be removed from that list.
-//! - `EraValidators::non_reserved`: validators that can be kicked out from that list
+//! - `EraValidators::non_reserved`: validators that can be banned out from that list
 //!
-//! # Kick out logic
+//! # Ban logic
 //! In case of insufficient validator's uptime, we need to remove such validators from
 //! the committee, so that the network is as healthy as possible. This is achieved by calculating
 //! number of _underperformance_ sessions, which means that number of blocks produced by the
 //! validator is less than some predefined threshold.
 //! In other words, if a validator:
 //! * performance in a session is less or equal to a configurable threshold
-//! `CommitteeKickOutConfig::minimal_expected_performance` (from 0 to 100%), and,
-//! * it happened at least `CommitteeKickOutConfig::underperformed_session_count_threshold` times,
-//! then the validator is considered an underperformer and hence removed (ie _kicked out_) from the
+//! `BanConfig::minimal_expected_performance` (from 0 to 100%), and,
+//! * it happened at least `BanConfig::underperformed_session_count_threshold` times,
+//! then the validator is considered an underperformer and hence removed (ie _banned out_) from the
 //! committee.
 //!
 //! ## Thresholds
-//! There are two kick-out thresholds described above, see [`CommitteeKickOutConfig`].
+//! There are two ban thresholds described above, see [`BanConfig`].
 //!
 //! ### Next era vs current era
-//! Current and next era have distinct thresholds values, as we calculate kicks during elections.
+//! Current and next era have distinct thresholds values, as we calculate bans during elections.
```

Comment:
De we want to add a section here about two new election modes?

Reply:
Added.